packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -0,0 +1,529 @@
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Installation
+
+You’ll need to download a few things before you can get started designing UIs with Paste.
+
+### Abstract
+
+All product designers across Twilio use [Abstract](https://www.abstract.com) to host and version control
+their design files. We also use Abstract to link Paste Sketch libraries and manage library updates (more on
+that later). Connect with your design manager to get added to the Twilio team in Abstract.
+
+<Callout variant="information">
+  <CalloutTitle>
+    Getting familiar with Abstract
+  </CalloutTitle>
+  <CalloutText>
+    If you are not yet familiar with Abstract, please take some time to learn
+    about <Anchor href="https://www.abstract.com/how-it-works">how it works</Anchor>,
+    and read through these resources
+    on <Anchor href="https://help.abstract.com/best-practices/">best practices</Anchor>.
+  </CalloutText>
+</Callout>
+
+### Sketch
+
+Product designers across Twilio use Sketch to design their UIs. You can [file a ticket with IT](https://twilio.service-now.com/sp/)
+to get a Sketch license.
+
+### Font files
+
+#### Fira Mono
+
+If you are using <strong>any theme</strong> in Paste, you will need
+to [download and install Fira Mono](https://fonts.google.com/specimen/Fira+Mono?query=fira+mono),
+our monospace typeface for code. It’s a free Google font.
+
+#### SF Pro
+If you are using <strong>any theme</strong> in Paste, you will need to [download and install SF Pro](https://developer.apple.com/fonts/), the
+system font for Apple platforms. We represent native web element styles (like option lists in [Select](/form-elements/select/)) in
+our design assets in the macOS styles.
+
+#### Inter
+
+If you are using the <strong>Default theme</strong> in Paste, you will need
+to [download and install Inter](https://github.com/rsms/inter/releases/download/v3.15/Inter-3.15.zip).
+It’s a free font. After you download the files, follow the instructions in the
+`install-mac.txt` file in your download. We recommend following the steps using Font Book.
+
+#### Whitney
+
+If you are using the <strong>Console theme</strong> in Paste, you will need to download Whitney.
+Reach out to us in `#help-design-system` if you need access to the Whitney font
+files for your work.
+
+## How to use Sketch libraries
+
+For every Abstract project, you
+will need to link the necessary Paste Sketch libraries in order to use the
+Sketch symbols, text styles, and layer styles.
+
+<Callout variant="information">
+  <CalloutTitle>
+    Tips on linking libraries in Abstract
+  </CalloutTitle>
+  <CalloutText>
+    If you haven’t linked libraries in Abstract before, follow <Anchor href="https://help.abstract.com/libraries/link-libraries/">this guide</Anchor> to learn how.
+  </CalloutText>
+</Callout>
+
+### Paste components
+
+<Anchor>
```

Comment:
A tooltip component must wrap the tooltip trigger. Because the trigger doesn't go anywhere, it shouldn't be an anchor. It must be a button. 

You can remedy this just by removing the outer `Anchor`, and adding `role="button"` to the `Box`. The tooltip component is making the box focusable.

Reply:
this applies to all of these status things.

Reply:
updated! lmk if those look correct now